.travis.yml

```diff
@@ -26,7 +26,7 @@ before_install:
 - # ./scripts/build_openssl.sh
 - sudo add-apt-repository ppa:tah83/secp256k1 -y
 - sudo apt-get update
-- sudo apt-get install libsecp256k1-dev
+- sudo apt-get install libsecp256k1-dev --allow-unauthenticated
```

Comment:
I'm sure this option was put in intentionally, but why is libsecp256k1-dev suddenly not authenticated?

Reply:
This seems like an intermittent issue (likely with the Canonical key server) because some builds had this error and some didn't. I added them anyway since it's fine for us.